src/cli.ts

```diff
@@ -96,8 +102,9 @@ const cwd = process.cwd();
  */
 const sourcePath = normalizePath(argv.flags.src, true);
 const distPath = normalizePath(argv.flags.dist, true);
+const tsconfig = parseTsconfig(argv.flags.tsconfig);
```

Comment:
While `parseTsconfig` should be used if a tsconfig file is passed in, it should default to `getTsconfig` if it's not passed in.

For reference, it should look like this:
https://github.com/esbuild-kit/esm-loader/blob/332a0a0220dbb80de806ebb3a15ec7c3ce893bee/src/utils.ts#L13-L20